packages/compartment-mapper/test/fixtures-cjs-compat/node_modules/app/_generateExpected.mjs

```diff
@@ -0,0 +1,16 @@
+// run me with:
```

Comment:
this file doesn't run as a test. I can clean it up if documenting how I got the result is not worth keeping a separate file. 

Couldn't decide.

Reply:
I like this here. The comment helps. More comment would be awesome.

Reply:
The underscore in the name is unnecessary to my eye.

Reply:
And I personally favor hyphens over camels in paths and URLs.

Reply:
`_` is there to indicate the file is not part of the fixture. After years of using almost opposite conventions for work and private projects I just agree to anything, so consider it done.

 I'm upset that there's a file to generate this, but the other example is only documented with a comment in index.js - I'll unify that one way or the other.